src/main/java/animals/service/FileService.java

```diff
@@ -0,0 +1,45 @@
+package animals.service;
+
+import animals.model.Node;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.File;
+import java.io.IOException;
+
+public abstract class FileService {
+
+    public void save(Node root) {
+        String fileName = getFileName();
+        ObjectMapper objectMapper = getObjectMapper();
+        try {
+            objectMapper
+                    .writerWithDefaultPrettyPrinter()
+                    .writeValue(new File(fileName), root);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Currently if we can't save to file, we print error to console and continue the program.
It doesn't look correctly. We should interrupt program in this case.